doc/start/hardware-recommendations.rst

```diff
@@ -249,30 +253,32 @@ performance of sequential and random reads and writes.
 
 Relatively inexpensive SSDs may appeal to your sense of economy. Use caution.
 Acceptable IOPS are not the only factor to consider when selecting SSDs for
```

Comment:
By the way, should this one be "is" and not "are"?

Reply:
Gray area, but you're right.